### PR TITLE
Update sample to use aspnetcore.dll and aspnetcorerh.dll

### PR DESCRIPTION
**Pull request by jkotalik**
I am currently hard coding this to x64. I don't believe $Platform would work. @natemcmaster is there a way to determine the bitness of the published application?

### CODE REVIEW

**Comment by natemcmaster** [comment]
> is there a way to determine the bitness of the published application?

This sample is a portable .NET Core app, so CPU arch cannot be determined until runtime.

You could restrict the sample to win-x64 by adding `<RuntimeIdentifier>win-x64</RuntimeIdentifier>` to the csproj.

File: samples/NativeIISSample/NativeIISSample.csproj [hunk 1]
```diff
@@ -5,7 +5,18 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.IISIntegration\Microsoft.AspNetCore.Server.IISIntegration.csproj" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+    <!-- x86 -->
+    <None Include="..\..\src\RequestHandler\bin\$(Configuration)\Win32\aspnetcorerh.dll" CopyToOutputDirectory="PreserveNewest" Visible="false" Link="x86\aspnetcorerh.dll" />
```
Comment (natemcmaster) [inline samples/NativeIISSample/NativeIISSample.csproj]: Does the global shim pickup `<app dir>/{x86,x64}/aspnetcorerh.dll`?

Comment (jkotalik) [reply samples/NativeIISSample/NativeIISSample.csproj]: It will look in the directory that aspnetcore.dll is loaded from for the requesthandler.

Comment (natemcmaster) [reply samples/NativeIISSample/NativeIISSample.csproj]: 👍 cool. thanks for clarifying

**Pull request merged**